Adding api to accept promise (#11)
freshworks/ember-sequential-render: tests/dummy/app/templates/docs/usage.md

In this example, the critical or hero element is the content inside the left pane. So that'll be assigned a renderPriority of 0. Subsequently, the quick notes and the participants list will be assigned a renderPriority of 1.

> ## 4. Wrap the content in sequential-render instances and pass in the function which performs async operation to get data by returning promise object

Reviewer: ```suggestion
## 4. Wrap the content in sequential-render instances and pass in a function that returns a promise.
```
